apps/dot_ext/forms.py

```diff
@@ -59,7 +59,11 @@ def __init__(self, user, *args, **kwargs):
         self.fields["name"].label = "Name*"
         self.fields["name"].required = True
         self.fields["client_type"].label = "Client Type*"
+        self.fields["client_type"].required = False
         self.fields["authorization_grant_type"].label = "Authorization Grant Type*"
+        self.fields["authorization_grant_type"].required = False
+        authorization_grant_type = self.fields["authorization_grant_type"]
+        authorization_grant_type.widget = authorization_grant_type.hidden_widget()
```

Comment:
I think you only need to make the fields optional and cut lines 65 and 66

Reply:
Yes, I cleaned up and pushed the code.